core/src/test/scala/org/apache/spark/sql/delta/expressions/InterleaveBitsSuite.scala

```diff
@@ -98,4 +158,26 @@ class InterleaveBitsSuite extends SparkFunSuite with ExpressionEvalHelper {
     InterleaveBits(Seq(Literal(0.toDouble))).checkInputDataTypes() != TypeCheckSuccess
     InterleaveBits(Seq(Literal(0.toString))).checkInputDataTypes() != TypeCheckSuccess
   }
+
+  test("randomization interleave bits") {
+    val numIters = sys.env
+      .get("NUMBER_OF_ITERATIONS_TO_INTERLEAVE_BITS")
+      .map(_.toInt)
+      .getOrElse(1000000)
+    var i = 0
+    while (i < numIters) {
```

Comment:
you are not using a specific random seed. if there is a failure, it will be very hard for any one repro that scenario. So pick a random seed, and print it along with the error message

Reply:
The input to the verification failure is already printed with the message.

Reply:
okay. lets make sure output with mkString actually prints out the int array correctly.